.github/workflows/artifact-tests.yml

```diff
@@ -1,13 +1,20 @@
 name: artifact-unit-tests
-on: push
+push:
+  branches:
+    - master
+  paths-ignore:
+    - '**.md'
+pull_request:
+  paths-ignore:
+    - '**.md'
 
 jobs:
   build:
     name: Build
 
     strategy:
       matrix:
-        runs-on: [ubuntu-latest, windows-latest, macOS-latest]
+        runs-on: [ubuntu-latest, windows-latest, macos-latest]
```

Comment:
All of our public documentation has `macos` so I'm changing it just for consistency

https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources

https://github.com/actions/virtual-environments#available-environments

Reply:
These are case-insensitive and `macOS` is more correct than `macos` based on how Apple names it